README.md

```diff
@@ -40,10 +40,19 @@ async function handleEvent(event) {
     try {
       return await getAssetFromKV(event, { mapRequestToAsset: customKeyModifier })
     } catch (e) {
-      return new Response(`"${customKeyModifier(event.request.url)}" not found`, {
-        status: 404,
-        statusText: 'not found',
-      })
+      if (resp instanceof KVError) {
+        switch (e.code) {
+          case 404:
+            return notFoundResponse
+            break
+          case 405:
+            return methodNotAllowedResponse
```

Comment:
```suggestion
            return new Response(`method ${event.request.method} not allowed`}
```

Reply:
ingMeh I could go either way on this but I think since it's the README, keep this line short and sweet prevents overwhelming the reader with an already complex example

Reply:
i do appreciate the want for simplicity; however it might make it seem like `methodNotAllowedResponse` is a pre-defined thing available to the user.

Reply:
This is just the README it's not meant to be functional, even copy pasteable code but a guide to seeing what the function does on a high level

Reply:
people copy paste code out of readmes

Reply:
I disagree. Maybe we can address this is a separate issue? I see other areas to improve the readme as well that I don't want to block this

Reply:
there are a lot of ways to improve the readme, but as it is right now, all the code examples in this readme are functioning code.

Reply:
The current snippet didn't actually function. I removed the specifics on handling the errors and it now functions